README.md

```diff
@@ -53,32 +53,38 @@ You can just emit actions directly if you want:
 from aviary.core import ToolCall
 
 
-def get_asv(self, agent_state, obs):
+def get_asv(agent_state, obs):
     action = ToolCall.from_name("calculator_tool", x="3 * 2")
     return action, agent_state, 0
 ```
 
-but likely you want to do something more sophisticated. Here's how our `SimpleAgent` - which just relies on a single LLM call - works (typing omitted):
+but likely you want to do something more sophisticated.
+Here's how our `SimpleAgent` - which just relies on a single LLM call - works (typing omitted):
 
 ```py
-from ldp.graph import compute_graph
+from ldp.agent import Agent
+from ldp.graph import LLMCallOp
 
 
 class AgentState:
-    def __init__(messages, tools):
+    def __init__(self, messages, tools):
         self.messages = messages
         self.tools = tools
 
 
 class SimpleAgent(Agent):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.llm_call_op = LLMCallOp()
 
     async def init_state(self, tools):
         return AgentState([], tools)
 
-    @compute_graph()
```

Comment:
We don't need this for a minimal example

Reply:
Can you change [this](https://github.com/Future-House/ldp/blob/main/ldp/alg/rollout.py#L303) to `OpResult.unwrap_value(action)` then? We are assuming the return of `get_asv` is `OpResult[ToolRequestMessage]`

Reply:
Oops ignore me - forgot that an Op will make a compute graph if not already in one.

Reply:
> forgot that an Op will make a compute graph if not already in one.

Ah I see, and dang. In our other `Agent`s we avoid manipulating the graph's `LLMCallOp`'s prior inputs with this:

```python
next_state.messages = [*next_state.messages, result.value]
```

However, this `README`'s `SimpleAgent` doesn't have that, and I was trying to avoid that potential bug by removing the `compute_graph` part. But I guess it's unavoidable, so for now we can kick this can down the road

Reply:
If you have any ideas here, lmk